packages/elements/src/components/ino-fab/ino-fab.tsx

```diff
@@ -122,14 +130,17 @@ export class Fab implements ComponentInterface {
       'mdc-fab--mini': this.inoMini,
     });
 
+    const iconSlotHasContent = hasSlotContent(this.el, 'ino-icon-leading');
+
     return (
       <Host id={this.uniqueHelperId}>
         <button class={classFab} disabled={this.inoDisabled}>
           <span class="material-icons mdc-fab__icon">
-            {this.inoIcon ? (
-              <ino-icon class="mdc-button__icon" ino-icon={this.inoIcon} />
```

Comment:
The `mdc-button__icon` class has been removed intentionally as it only decreases the size of the icon by 1px, which is insignificant. It has no noticeable effect on the appearance of the `ino-fab` component.

Reply:
good to know, thanks for the heads-up